### DISCUSSION TITLE
Star option for RNA-seq mapping to determine C-To-U edit

### DISCUSSION THREAD

**Discussion by CelineLabbeCurie** [alexdobin/STAR#1948] [Q&A]
Hello,

We've done RNA-seq after transfection of APOBEC coupled with or without other proteins and we want to analyse the differences in C-To-U edition between conditions.

I've done the mapping on Hg38 with this options :

--outMultimapperOrder Random
--outSAMtype BAM SortedByCoordinate 
--outSAMprimaryFlag OneBestScore 
--outFilterType BySJout 
--outFilterMatchNmin 16
--outFilterMultimapNmax 1 
--outFilterMismatchNmax 999
--outFilterMismatchNoverLmax 0.07
--alignIntronMin 20
--alignIntronMax 1000000
--alignMatesGapMax 1000000
--alignSJoverhangMin 8 
--alignSJDBoverhangMin 1           

Some people told me that the editing could be underestimated.

Does anybody have already done something like this ? Do I need to change the options for the mapping ? Someone told me that, perhaps, I should put "N" instead of all the C otherwise I will have a lot of mismatch, but it's genome wide so that would be a lot a base changes. Any thought on that ? 
I'm cleary not an expert here. I've some other posts on STAR options but I'm still unsure on what to do in this case.

I welcome any help.
Thanks a lot.
Best

**Comment by alexdobin** [comment]
Hi Celine,

what is a typical distance between two edit events? If they are too close (<20-30b), mapping could be problematic.
One approach that is commonly used for bisulfite sequencing analysis is to replace C->T for all C bases in the genome and reads.

**Comment by CelineLabbeCurie** [comment]
Hi Alex,

Thanks for answering me.

The editing can indeed be less then 20-30b as all C will be edited in the region so it depend of the number of C in the area. 
Do you think I should change the option "outFilterMismatchNoverLmax" ?

I'll try to change the base as suggested. Maybe it can be done only on the unmapped reads ?

Best

**Comment by alexdobin** [comment]
For high-density editing, I would recommend the alphabet reduction methods as used in bisulfite sequencing.
